doc/COMPILE.md

```diff
@@ -712,6 +712,32 @@ For reference, you can look into the
 and the
 [CMake in OpenWRT](https://github.com/openwrt/openwrt/blob/master/include/cmake.mk).
 
+### Resolving Missing \*.so Library Error
+
+In case you fail to run Elektra with the message like this one
+`Reason: of module: libelektra-resolver.so, because: libelektra-resolver.so: cannot open shared object file: No such file or directory`
+you have 2 options to solve this problem.
+
+Option 1:
+Run this script to set environmental variables for development purpose:
+
+```sh
+. /home/osboxes/TU/libelektra/scripts/run_dev_env
```

Comment:
This is specific to your config, it will not help someone else.

Reply:
Ok, I will remove it.

Reply:
When you write: "Let us assume that the source dir is in..., then..." it would be okay.